src/jmcomic/cl.py

```
@@ -1,9 +1,11 @@
 import warnings
+
 from .cli import *
```

Ruff (0.16.0)

[error] `from .cli import *` used; unable to detect undefined names

(F403)